README.md

```diff
@@ -56,24 +57,42 @@ This exercise takes you through the steps to use InterSystems IRIS multi-model c
 	**Note**: if you are using python 2 or earlier, follow the commented instructions in `createschema.py` in the `connect_to_iris()` function to configure the connection properly.
 
 3. Run `python createSchema.py`. 
-	* **Note**: This exercise is configured for Python 3. For some users, you may need to run `python3 createSchema.py` if the `python` command defaults to Python 2. 
+   * **Note**: This exercise is configured for Python 3. For some users, you may need to run `python3 createSchema.py` if the `python` command defaults to Python 2. 
 	
-1. Open the management portal by following the link given to you when you created your instance of the InterSystems IRIS learning labs (or if on the docker container, go to [http://localhost:52773/csp/sys/%25CSP.Portal.Home.zen](http://localhost:52773/csp/sys/%25CSP.Portal.Home.zen), navigate to **System Explorer > SQL** and expand the **Tables** section.  Observe that the Demo.Employee table has been created.
-## Modify the table class using InterSystems ObjectScript
+4. Validate the `Demo.Employee` table has been created.
+   1. Open the management portal by following the link given to you when you created your instance of the InterSystems IRIS sandbox (or if on the docker container, go to [http://localhost:52773/csp/sys/%25CSP.Portal.Home.zen](http://localhost:52773/csp/sys/%25CSP.Portal.Home.zen). 
+   2. Switch to the `USER` namespace, if necessary, by clicking "Switch" next to "Namespace %SYS"
+   3. Navigate to **System Explorer > SQL** and expand the **Tables** section. Find `Demo.Employee` in the list.
 
-### Setting Up Atelier
 
-1. If you have not installed InterSystems Atelier, follow [these instructions](https://download.intersystems.com/download/atelier.csp). Once downloaded and installed, open Atelier and connect your InterSystems IRIS instance to it.  
+## Modify the table class using InterSystems ObjectScript
 
-	Atelier allows you to edit InterSystems IRIS classes directly so that you can customize how they behave. When you ran `createSchema.py` earlier, InterSystems IRIS automatically created an ObjectScript class that represents that table.  We will need to modify this class to enable it to receive JSON data.
+### Setting Up the Visual Studio Code ObjectScript Extension
+
+1. If you have not installed InterSystems Visual Studio Code, do so  [here](https://code.visualstudio.com/).
+2. Open Visual Studio code. In the extensions manager, search for and install the `InterSystems ObjectScript` extension.
+3. Select a folder for your workspace by selecting **Open Folder** in the **File Explorer**, or **File** > **Open**. Navigate to the folder where you .
```

Comment:
The 'Open Folder' option is distinct from the File> Open workflow. If you open a new window in VSCode it will prompt you to select a folder (or clone a repo) for that workspace: 
![image](https://user-images.githubusercontent.com/54324676/81828751-274f3200-9508-11ea-8940-1abe02ddd559.png)